.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -234,24 +234,31 @@ on:
           - eltwise.binary.multiply.multiply_scalar_pytorch2
           - eltwise.binary.div.div
           - eltwise.binary.div.div_tensor_pytorch2
+          - eltwise.binary.div.div_forge
           - eltwise.binary.div_no_nan.div_no_nan
           - eltwise.binary.logical_or.logical_or_
           - eltwise.binary.logical_or.logical_or
           - eltwise.binary.logical_or.logical_or_output
+          - eltwise.binary.logical_or.logical_or_forge
           - eltwise.binary.logical_xor.logical_xor_
           - eltwise.binary.logical_xor.logical_xor
           - eltwise.binary.logical_and.logical_and_
           - eltwise.binary.logical_and.logical_and
           - eltwise.binary.logical_and.logical_and_output
+          - eltwise.binary.logical_and.logical_and_forge
           - eltwise.binary.polyval.polyval
           - eltwise.binary.remainder.remainder
+          - eltwise.binary.remainder.remainder_scalar_pytorch2
```

Comment:
Where is the remainder_scalar_pytorch2 file located?

Reply:
It is also under binary/remainder/
`tests/sweep_framework/sweeps/eltwise/binary/remainder/remainder_scalar_pytorch2.py`
I simply grouped binary remainder tests together.